etl/steps/data/garden/faostat/2023-02-22/additional_variables.py

```diff
@@ -636,6 +636,71 @@ def generate_macronutrient_compositions(df_fbsc: pd.DataFrame) -> Table:
     return tb_combined
 
 
+def generate_fertilizers(df_ef: pd.DataFrame, df_rl: pd.DataFrame) -> Table:
+    # Item code for "Cropland" (which includes arable land and permanent crops).
+    ITEM_CODE_FOR_CROPLAND = "00006620"
+
+    # Element code for element "Area" of faostat_rl dataset.
+    ELEMENT_CODE_FOR_AREA = "005110"
+
+    # Item codes for fertilizers in faostat_ef (namely nitrogen, phosphate and potash).
+    ITEM_CODES_FOR_FERTILIZERS = ["00003102", "00003103", "00003104"]
+
+    # Element code for use per area of cropland.
+    ELEMENT_CODE_FOR_USE_PER_AREA = "005159"
+
+    # Convert units from kilograms to tonnes.
+    KG_TO_TONNES = 1e-3
+
+    # Select necessary element (use per area).
+    fertilizers = df_ef[(df_ef["element_code"] == ELEMENT_CODE_FOR_USE_PER_AREA)].reset_index(drop=True)
+
+    # Sanity checks.
+    error = "Unit for use per area has changed."
+    assert list(fertilizers["unit"].unique()) == ["kilograms per hectare"], error
+
+    error = "Unexpected list of item codes for fertilizers (maybe another was added to faostat_ef)."
+    assert set(fertilizers["item_code"]) == set(ITEM_CODES_FOR_FERTILIZERS), error
+
+    # Transpose fertilizers data.
+    fertilizers = fertilizers.pivot(index=["country", "year"], columns=["item"], values=["value"]).reset_index()
+    fertilizers.columns = ["country", "year", "nitrogen_per_cropland", "phosphate_per_cropland", "potash_per_cropland"]
```

Comment:
Hey Lucas, you are absolutely right, that was also something I wasn't happy about. But there's some issue with using `pivot` that changes the index names that I couldn't figure out how to avoid. I'll merge this PR (since ETL is [failing in master](https://owid.slack.com/archives/C0193RW5E2J/p1681821424242289)) and change the way I rename columns in a different one.